vpc/proto/vpc.proto

```diff
@@ -356,6 +356,13 @@ message DetachBranchNetworkInterfaceResponse {
     string associationID = 3;
 }
 
+message ListBranchToTrunkENIMappingRequest {
+}
+
+message ListBranchToTrunkENIMappingResponse {
+    map<string, string> branchENIMapping = 1;
```

Comment:
Might be work adding a comment saying that this is key of branch ENI -> trunk ENI.